### PR TITLE
Add mozza event parquet output

### PR DESCRIPTION
**Pull request by fbertsch**
:whd, just want to make sure that d2p for moz_ingest will work similarly. It seems all of those metadata fields are available. Feel free to reassign reviewers if you're busy.

### CODE REVIEW

File: schemas/mozza/event/event.1.parquet.txt [hunk 1]
```diff
@@ -0,0 +1,29 @@
+message event {
+  required group metadata {
+    required binary documentId (UTF8);
+    required int64  Timestamp;
+    required binary Date (UTF8);
```
Comment (whd) [inline schemas/mozza/event/event.1.parquet.txt]: This is a header specified by the client so will only be available if the client sends it.

**Comment by fbertsch** [comment]
@whd I think that's okay for our purposes, we control the client so we should ensure it's available. RFAL

File: templates/include/common/event.1.parquetmr.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+(TUPLE) {
```
Comment (trink) [inline templates/include/common/event.1.parquetmr.txt]: Not liking the partial object template unfortunately the other options aren't much better with this simple templating.

I would get rid of the extra indent spacing (just go with the standard 2) as it adds little value and won't align properly in the two different contexts where the template is used.  Added #105 to address formatting in general.

File: schemas/telemetry/focus-event/focus-event.1.parquetmr.txt [hunk 3]
```diff
@@ -39,6 +39,7 @@ message focus_event {
           }
         }
       }
+
```
Comment (trink) [inline schemas/telemetry/focus-event/focus-event.1.parquetmr.txt]: We discussed cleaning this up with `string(STRIP "${${VARNAME}}" ${VARNAME})` but I would rather have it addressed by #105

**Comment by fbertsch** [comment]
@trink I'll leave the pretty-printing for later. Changed the indentation, RFAL

**Pull request merged**